WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustoms.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import WooFoundation
+
+struct WooShippingCustoms: View {
+    let informationIsCompleted: Bool
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        HStack {
```

Comment:
What do you think of using `AdaptiveStack` instead of `HStack` here, so the layout adjusts with larger dynamic font sizes?

Reply:
Good idea! Done in https://github.com/woocommerce/woocommerce-ios/pull/14729/commits/83753cab2d5fe3418ea2ed4babc471adbf4744a3